element_deeplabcut/readers/dlc_reader.py

```diff
@@ -9,16 +9,18 @@
 class PoseEstimation:
 
     def __init__(self, dlc_dir=None, pkl_path=None, h5_path=None, yml_path=None,
-                 filename_prefix=''):
-        if dlc_dir is None:
+                 filename_prefix='', output_dir=None):
```

Comment:
Perhaps it was a bit unclear, but `dlc_dir` here is meant to be the same as the `output_dir` you just added - i.e. the directory containing the pose estimated results.
We don't need another `output_dir`, perhaps we can rename `dlc_dir` to `output_dir`, but to me `output_dir` is this context is vague and ambiguous as well. I'm happy to consider a better term here.